src/assets/simp-adapter/build/simp-adapter.spec

```diff
@@ -36,8 +36,6 @@ Summary: SIMP Adapter for the Puppet Enterprise Puppet Installation
 License: Apache-2.0
 Requires: rsync
 Requires(post): puppet-agent
-Requires(post): pe-puppetserver
```

Comment:
Looking at diff instead of the full file, led to this bad comment.  I do think retaining the Requires(post) would help with the race condition in changing the puppet gid/uid for puppetserver files.  In this case, we would want a corresponding Requires(post) for the simp-adapter-foss RPM, as well.